.gitignore

```diff
@@ -1,2 +1,4 @@
 .idea/
 *.pyc
+db.sqlite3
+project-env
```

Comment:
didn't you forget to add `__pycache__` there?

Reply:
I actually did, Thank you for an advice.